### PR TITLE
Allow including js in head section

### PR DESCRIPTION
**Pull request by jladage**
Fixes https://github.com/plone/Products.CMFPlone/issues/3931

### CODE REVIEW

File: src/plone/base/interfaces/controlpanel.py
Comment (stevepiercy) [inline src/plone/base/interfaces/controlpanel.py:1201]:
```suggestion
            "For enabling third-party JavaScript integrations "
```

File: src/plone/base/interfaces/controlpanel.py
Comment (stevepiercy) [inline src/plone/base/interfaces/controlpanel.py:1202]:
```suggestion
            "from external providers (e.g., Google "
```

File: src/plone/base/interfaces/controlpanel.py
Comment (stevepiercy) [inline src/plone/base/interfaces/controlpanel.py:1203]:
```suggestion
            "Analytics), paste the provided code snippet here. "
```

File: src/plone/base/interfaces/controlpanel.py
Comment (stevepiercy) [inline src/plone/base/interfaces/controlpanel.py:1214]:
```suggestion
            "For enabling third-party JavaScript integrations "
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor spelling and grammar corrections.

This also needs a change log entry.

**Comment by stevepiercy** [comment]
@jladage see [Change log entry](https://6.docs.plone.org/contributing/index.html#change-log-entry).

I don't know why lint fails in CI.

The pre-commit.ci caught the same spelling error that I caught.

**Comment by jladage** [comment]
@jenkins-plone-org please run jobs

**Comment by jladage** [comment]
@stevepiercy All checks now pass, but merging remains blocked. Should you do something to accept the change request? Sorry, I lack some experience with these features of github. ;)

**Comment by stevepiercy** [comment]
@jladage in the future, include "Fixes #[issue number] in your PR description. This enables automatic closing of the related issue when the pull request is merged. This also creates a hyperlink to the original issue for easy reference. I edited your description accordingly.

**Review by stevepiercy — APPROVED** [review]
LGTM, but a core contributor should review and merge. I just triage stuff.

**Review by mauritsvanrees — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
@erral FYI After merging, this would need an update of `plone.app.locales` for two updated messages and two new ones.

**Review by gforcada — APPROVED** [review]
Nice! I was looking for something similar 😄

**Review by jensens — APPROVED** [review]

**Comment by jensens** [comment]
Does this need a migration in plone.app.upgrade?

**Comment by mauritsvanrees** [comment]
> Does this need a migration in plone.app.upgrade?

Yes, this is already there: https://github.com/plone/plone.app.upgrade/pull/323

**Review by thet — COMMENTED** [review]
+/-0, as I do not have the use case (yet) for this and I personally would add JavaScript differently. However, one minor remark: why would we name this `webstats_head_js` if we allow anything? Okay, there is this security check in https://github.com/plone/plone.app.layout/pull/361 but I doubt this is necessary or useful at all, see my remarks in the other PR.

File: src/plone/base/interfaces/controlpanel.py
Comment (jladage) [reply src/plone/base/interfaces/controlpanel.py:1198]: Well, since the existing is named webstat_js and I didn't want to change that fields name I came up with this fieldname to communicate their relation. If we name this one inject_head, than would you rename webstats_js to inject? That doesn't make sense to me, sorry. ;)

In an ideal world it would be refactored to names like inject_javascript and inject_javascript_head. I hope other reviewers can live with it being this way.

Comment (thet) [reply src/plone/base/interfaces/controlpanel.py:1198]: fair enough

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**
